.github/workflows/multi-arch-test-build.yml

```diff
@@ -95,17 +95,60 @@ jobs:
       - name: Move created packages to project dir
         run: cp bin/packages/${{ matrix.arch }}/packages_ci/*.ipk . || true
 
+      - name: Collect metadata
+        run: |
+          VERSION=${BRANCH#openwrt-}
+          if [ $VERSION = master ]; then VERSION=99.99+master; fi
```

Comment:
The 99.99 is a bit confusing, shouldn't we just keep it as master/snapshot?

Reply:
I was looking for ways to make consumption of this data easy for users, and the python PKG-INFO format specification ( is strict about at least having a few numbers in there,  However, I did not check what the `pkginfo` tool requires, it might not be as strict. I'll check that soon, because I also prefer just sticking `master` or `snapshot` in there.

Reply:
Ok, it's not picky at all, so I'll remove all processing of $BRANCH and just stick it into Version directly.